cmd/tarmak/cmd/cluster.go

```diff
@@ -31,6 +31,20 @@ func clusterApplyFlags(fs *flag.FlagSet) {
 		false,
 		"apply changes to infrastructure only, by running only terraform",
 	)
+
+	fs.BoolVar(
+		&store.AutoApprove,
+		"auto-approve",
+		false,
```

Comment:
This is only for queries generated by tarmak, not terraform

Reply:
If I run tarmak cluster apply before your PR it will just apply the stuff without any interactive query
After the PR it will query to continue. This will break jenkins automation in babylon and news and we should avoid that
So the default should be not to query and just apply, in a later version we can swap the defaults